index.bs

```diff
@@ -206,6 +206,27 @@ Modify the step:
 
 to call <a spec="html">navigate</a> with |attributionSource| set to |attributionSource|.
 
+<h3 id="monkeypatch-window-open">Window open steps</h4>
+
+Attribution source information declared via `window.open()` need to be passed to
+the <a spec="html">navigate</a> algorithm.
+
+In <a spec="html">window open steps</a> after
+
+> 10. If target browsing context is null, then return null.
+
+add the following steps:
+1. Let <var>attributionSource</var> be the result of running [=obtain an attribution source from window features=] with tokenizedFeatures and source browsing context
+
+Modify the step:
+> 3. Navigate target browsing context to request, with exceptionsEnabled set to true and the source browsing context set to source browsing context.
```

Comment:
We should also add the cross-reference for `exceptionsEnabled` and `the source browsing context` as in the spec itself.

Reply:
Done